examples/demo_bgp.pp

```diff
@@ -19,7 +19,29 @@
   # --------------------------------------------------------------------------#
   # Configure Global BGP                                                      #
   # --------------------------------------------------------------------------#
-  cisco_bgp { '55.77 blue':
+
+  $maxas_limit = $operatingsystem ? {
+    'nexus' => '50',
+    default => undef
```

Comment:
We should see that for all of them, actually - setting an unsupported property to 'default' should be a no-op, IIRC. Might be worth trying that in the demo manifest and/or the beaker tests, come to think of it.